decisioncentral-indexing/image.yaml

```diff
@@ -48,7 +48,7 @@ modules:
           - name: cct_module
             git:
                   url: https://github.com/jboss-openshift/cct_module.git
-                  ref: sprint-25
+                  ref: master
```

Comment:
shouldn't be sprint-26?

Reply:
There is no "sprint-26" yet in cct_module, so in image.yaml, we use "master". The reason cp-overrides.yaml and rel-overrides.yaml have "sprint-26" is because we are preparing those files for it.